config/locales/en.yml

```diff
@@ -20,6 +20,10 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
+  activerecord:
+    errors:
+      messages:
+        taken: 'must be unique within a provider context'
```

Comment:
is this default message tested anywhere?

Reply:
spec/features/collection_templates/template_creation_spec.rb 189